README.md

```diff
@@ -1 +1,47 @@
-# Vertiq
\ No newline at end of file
+# Vertiq.BTool
+
+**Vertiq.BTool** is a dotnet global tool
+that enables fast and efficient build script execution from any location on the commandline.
+
+## Description
+
+Whenever you need to locate and execute a build script within a repo subdirectory, Vertiq.BTool is there to help.
+Simply issue `b` command,
+and it will search the current directory and its subfolders for your build script, and execute it.
+No more navigating to build scripts!
+
+## Installation
+
+Download and install one of the currently supported [.NET SDKs](https://www.microsoft.com/net/download).
+Once installed, you can install the tool globally using the following command:
+
+```shell
+dotnet tool install --global vertiq.btool
+```
+
+### Update
+
+If you already have a previous version of **Vertiq.BTool** installed,
+you can upgrade to the latest version using the following command:
+
+```shell
+dotnet tool update --global vertiq.btool
+```
+
+### Uninstall
+
+If you want to remove the tool, you can do it using the following command:
+
+```shell
+dotnet tool uninstall --global vertiq.btool
+```
+
+## Usage
+
+```shell
+b
```

Comment:
we should issue that it's basically `b %*` so it's passing along its argument's to the called build script